### ISSUE TITLE
Suggestion : Completion Tooltip Options

### ISSUE THREAD

**Issue by deathb4retreat** [night/BetterTTV#2122]
One of the most frustrating things I've encountered regarding something through BTTV is getting so used to the Completion Tooltip and how simple that made everything. One of the things I've noticed bothers me most is when there's some very similar names, such as when watching a streamer named Zapman, and having somebody named Zapfan, it goes alphabetical order and always prioritizes Zapfan.

My idea, basically, is to *add options for Completion Tooltip*, so that if the streamer is available in the frame of what you typed (such as Z for Zapman) that the highest autocomplete would be the streamer. Other ideas include putting mods as higher than others, and finally friends/commonly "@ ed" people to the top of the completion tooltip. 

Basically just a Quality of Life option change, if possible. Thoughts?

**Comment by EhsanKia** [comment]
These are planned for BTTV v7, once we add advanced settings. Right now, as it is, cluttering the settings menu with small things like that is more counter productive than useful.

**Comment by deathb4retreat** [comment]
That's wonderful! As a relevant question to that, is there any guesstimate when BTTV v7 will be released? And if we have it through the Chrome store will it auto update? 

Thank you so much!

**Comment by thinkpad4** [comment]
I just hope that Completion Tooltip works like it currently works when I have the setting off.  I depend on it working the same when off
